src/com/reason/ide/settings/ReasonSettings.java

```diff
@@ -15,7 +15,7 @@
 
     private Project m_project;
     private boolean m_enabled = true;
-    private boolean m_refmtOnSave = true;
+    private boolean m_refmtOnSave = false;
```

Comment:
instead of defaulting the settings in two separate places, can we have the default constructor use an instance of ReasonSettingsState to initialize ReasonSettings?

something like:
```
// this _should_ be used when intellij creates this bean
public ReasonSettingsState() {
  ReasonSettingsState state = new ReasonSettingsState();
  this.m_refmtOnSave = state.refmtOnSave;
  ...
}
```